src/Classes/PassiveTree.lua

```diff
@@ -381,14 +361,6 @@ local PassiveTreeClass = newClass("PassiveTree", function(self, treeVersion)
 			if node.isAscendancyStart then
 				group.isAscendancyStart = true
 			end
-			if node.o ~= 4 then
```

Comment:
I think this is safe to remove since `self:ProcessNode` gets called a few lines down anyway and will do the same calculation